sys/include/net/gnrc/ipv6/nib.h

```diff
@@ -288,6 +288,19 @@ enum {
     GNRC_IPV6_NIB_ROUTE_INFO_TYPE_NSC,
 };
 
+/**
+ * @brief   Types for GNRC_NETIF_BUS_NIB per-interface message bus
+ */
+typedef enum {
+    /**
+     * @brief   Address registration successful
+     *
+     * The Address registration with the boarder router was successful.
+     * The node now has a global prefix.
```

Comment:
Address registration and prefix configuration are two distinct events. The prefix configuration actually happens simultaneously with the assignment of the address to the interface, so before address registration is even kicked off. Also, nodes don't have prefixes. Interfaces have.